### PR TITLE
Feature/project

### PR DESCRIPTION
**Pull request by shubham14p3**
The Weather App project is build using Asynchronous Javascript and APIs call with async/await feature keeping in mind.

## Project milestones

In this project, we learned using async/await, Asynchronous Javascript and APIs.

## Project Specifics:

##User Interaction UI
User can toggle between Fahrenheit/Celcius
Users can enter any valid location.
Users can get a live report.

## General requirements:
- [x] Add a descriptive README file to your repositories as described here.
- [x] Use meaningful names for your Pull Requests and add a short summary of its content as described here.
- [x] Use descriptive commit messages as described here.
- [x] Do not commit files that are not related to the project. Instead, add them to .gitignore file.
- [x] Do not commit inline comments. They will make your project look messy. If you need to review a previous version of your code, you can always use git history.

## Git Flow:
- [x] Use a specific branch for each feature.
- [x] Use a descriptive name for your branches.
- [x] Use Stylelint and ESlLint to validate your code.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+  ev.preventDefault();
+
+  // gets data from Open Weather API based on input location
+  weatherSearch();
+});
+
+temperatureUnitToggleButton.addEventListener(
+  'click',
+  toggleTemperatureUnitDisplay,
+);
```
Comment (bellom) [inline src/index.js:22]: we need to make different requests for the Celsius data and the Fahrenheit data.
image
https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59622133/

File: src/index.js [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+function toggleTemperatureUnitDisplay() {
+  if (fahrenheitSection.style.display === 'none') {
+    updateDisplayToFahrenheit();
+  } else {
+    updateDisplayToCelcius();
+  }
+
+  function updateDisplayToFahrenheit() {
+    temperatureUnitToggleButton.innerHTML = 'Switch to Celcius';
+    fahrenheitSection.style.display = 'block';
```
Comment (bellom) [inline src/index.js:157]: Let's separate the DOM from application logic, you can create another file for DOM.

image

https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59622133/

**Comment by shubham14p3** [comment]
Done all the changes, the link is working fine in Incognito, also I can't see any script present in y code.

**Comment by Forison** [comment]
> 1. All the link is working fine in Incognito/ Normal mode(Try clearing the cache),
> 2. I can't see any script present in my code can you confirm the above in package.json.
>    I have submitted for re-review do let me know if there is still any problem.

I hope our call answered all. Thank you

**Comment by shubham14p3** [comment]
Thanks for the review and making things clear, I have made all the necessary changes and everything is up and running, do let me know if you find anything else missing.

**Pull request merged**